firefox-ios/firefox-ios-tests/Tests/ClientTests/TrackingProtectionTests/BlockedTrackersTableViewControllerTests.swift

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import Common
+@testable import Client
+
+class BlockedTrackersTableViewControllerTests: XCTestCase {
+    let windowUUID: WindowUUID = .XCTestDefaultUUID
+    let blockedTrackersMockModel = BlockedTrackersTableModel(
+        topLevelDomain: "test.com",
+        title: "test.com",
+        URL: "test.com",
+        contentBlockerStats: TPPageStats(),
+        connectionSecure: true
+    )
+
+    override func setUp() {
+        super.setUp()
+        DependencyHelperMock().bootstrapDependencies()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        DependencyHelperMock().reset()
```

Comment:
nit: although this is prob a no-op method, good practice to add the call to super as the final cleanup
```suggestion
        DependencyHelperMock().reset()
        super.tearDown()
```